docs/source/conservedwaters.rst

```diff
@@ -1,8 +1,35 @@
-Theory, Background and Methods
-==============================
-
-What is a conserved water?
-What types of conserved waters exist?
-Where can we use this?
-Classification of conserved waters.
-See `here <https://doi.org/10.1021/acs.jcim.2c00801>`_.
\ No newline at end of file
+Theory, Background, and Methods
+===============================
+
+The developed method focuses on the identification and classification of conserved water molecules and their networks derived from molecular dynamics (MD) simulations. A unique feature of this method is its ability to not only determine the positions of oxygen atoms but also the orientations of hydrogen atoms. This dual focus enables a nuanced classification into types like FCW, HCW, and WCW, which is a significant advantage over existing methods. The classification provides vital information for assessing the water network's stability. The method operates solely on data from MD simulations, without requiring any crystal water data.
+
+
+Understanding Conserved Water
+-----------------------------
+
+Conserved water refers to water molecules that maintain specific, stable positions and orientations within biological systems, such as the active sites of proteins. These waters play pivotal roles in biochemical processes like protein-ligand binding and enzymatic reactions. Their presence and orientations significantly influence both the thermodynamics and kinetics of these processes.
+
+Types of Conserved Waters in Our Approach
+-----------------------------------------
+
+Conserved waters are categorized into three main types based on hydrogen atom orientations:
```

Comment:
I think you could also add a couple of sentences explaining how we treat the residence time and that this in conjuction with hydrogen orientation analysis gives a strong confidence that FCW and HCW have close to 100% residence time, while for WCW it might be debatable sometimes?